PokeAlarm/Twilio/TwilioAlarm.py

```diff
@@ -117,9 +117,18 @@ def raid_alert(self, raid_info):
 
     # Send a SMS message
     def send_sms(self, to_num, from_num, body):
-        args = {
-            'to': to_num,
-            'from_': from_num,
-            'body': body
-        }
-        try_sending(log, self.connect, "Twilio", self.__client.messages.create, args)
+        if isinstance(to_num, list):
```

Comment:
Could we reduce this? Instead of branching, can we do something like:
```python
if not isinstance(to_num, list):
     to_num = [to_num]
for num in to_num:
```
We could also do the check when we set in the number so we don't have to redo it for each message send.